webapp/src/components/table/table.scss

```diff
@@ -1,7 +1,7 @@
 .Table {
     margin-top: 16px;
     padding-bottom: 0 !important;
-    overflow: auto;
+    overflow: visible;
```

Comment:
I have some concerns about this, I have to test it manually.

Reply:
As I expected, this generate problems with the scrolling when the window is narrow or when the number of columns (or size of the columns) is high.

I would try to avoid that and find another way to show that without messing with the overflow there.

Reply:
@jespino, Thank you for the review and I understand why this overflow property should be auto, the only solution I can think about it right now is to increase the left padding of the table so that we can see the hidden column of the table on hover with keeping the table scrollable as well. Please let me know your thought. Thank you.

https://user-images.githubusercontent.com/26198062/156360741-d72f9a1d-c185-4c4e-a4e1-28bb08b41ae9.mp4

Reply:
Thank you for the review @jespino, I understand this didn't look nice, but this is the only solution possible I can think of with the current architecture of the code we are having because as Table have the overflow property as auto, it is not allowing the overflow div to be displayed on hover outside the table. 

If there is any other solution for it, please suggest, I can work on it. Thank you.

Reply:
@Rajat-Dabade I'll have a look at the code and may push a UI fix for this if I figure it out.

Reply:
@jespino @asaadmahmood, I have tried to solve the UI issue, and as far as I'm concerned it looks better than the previous one and moreover similar to the proposed UI design. I have pushed the changes, Can you please review them? Please let me know if there are any changes to be made. Thank you.

https://user-images.githubusercontent.com/26198062/157384305-cb6ad0b8-d382-4b14-b305-a53b91649f74.mp4